### ISSUE TITLE
Updating to 1.5 - Could not install package System.Text.Encoding.CodePages 4.3.0

### ISSUE THREAD

**Issue by chris-eaton** [dotnet/orleans#3230]
Hi all

I am updating to Orleans 1.5 and getting the following error when installing the nuget package, specifically the OrleansCodeGenerator package

> Could not install package 'System.Text.Encoding.CodePages 4.3.0'. You are trying to install this package into a project that targets '.NETFramework,Version=v4.6.1', but the package does not contain any assembly references or content files that are compatible with that framework. For more information, contact the package author	

At first I wondered if my nuget files were corrupt in some way, so I removed every single nuget package out of every project my entire solution and then re-added them manually [back to 1.4.2] and then tried to upgrade to 1.5 again and I get the same error.

Can someone help?

Notes:
- All my projects are .NET 4.6.1

**Comment by attilah** [comment]
Hi @chris-eaton!

I tried to repro it, by creating a new Console App targeting net 4.6.1 and adding the ```Microsoft.Orleans.CodeGenerator``` package and have not problem with it.

![image](https://user-images.githubusercontent.com/230432/28279024-03c17f02-6ad4-11e7-8282-4a70977f68e8.png)

What version of VS are you using? I tried VS2017 latest available version.

I've attached my solution to make you able to see if that works for you.

The mentioned package is a transitive reference for us, we're not using it directly.

Thanks,
Attila

[codegen-repro.zip](https://github.com/dotnet/orleans/files/1153358/codegen-repro.zip)
